src/oracles/UniV3Oracle.sol

```diff
@@ -29,20 +30,29 @@ contract UniV3Oracle is INFTOracle {
     /// @notice Oracle for price estimations
     IOracle public immutable oracle;
 
+    /// @notice Maximum sqrtPriceX96 deviation allowed between oracle and univ3 pool
+    uint256 public maxSqrtPriceX96Deviation;
```

Comment:
Not sqrt and not X96 anymore, `0.01 ether` should mean ~ 1% deviation between prices, as far as I understand

Maybe `maxPriceRatioDeviation` would be a better naming